iac-extensions/arm/src/test/java/org/sonar/iac/arm/parser/utils/Assertions.java

```diff
@@ -53,7 +53,7 @@ private void parseTillEof(String input) {
       TokenLocation loc = new TokenLocation(1, 0, input);
       if (!tree.textRange().end().equals(new TextPointer(loc.endLine(), loc.endLineOffset()))) {
         throw new RecognitionException(
-          0, "Did not match till EOF, but till line " + tree.textRange().end().line());
+          0, "Did not match till EOF, but till line " + tree.textRange().end().line() + " line offset: " + tree.textRange().end().lineOffset());
```

Comment:
I like it.

Reply:
No much pride on this, I just copy pasted it from Docker :grin: